skribenten-backend/src/main/kotlin/no/nav/pensjon/brev/skribenten/services/PenService.kt

```diff
@@ -56,6 +58,8 @@ interface PenService {
     data class KravStoettetAvDatabyggerResult(
         val kravStoettet: Map<String, Boolean> = emptyMap()
     )
+
+    suspend fun hentP1VedleggData(saksId: Long, spraak: LanguageCode): ServiceResult<Api.GeneriskBrevdata>
```

Comment:
Korfor trengs denne her, kan du ikkje berre bruke den vanlege hentPesysBrevdata?

Reply:
Åja, det er for å få vedleggsdata separat. Hmm.